### PR TITLE
feat(threaded-replies): fix styling for resolve

### PR DESCRIPTION
**Pull request by MateuszMamczarz**
Fix visual issues:
- adjust margin for `resolved` pill
- change fill for `X` icon to be consistent with other icons
Screen Shot 2022-11-07 at 18 22 48

### CODE REVIEW

**Review by wpiesiak — APPROVED** [review]

File: src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import '../../../common/variables';
+
+.bcs-AnnotationActivityMenu-unresolveAnnotation path {
+    fill: $bdl-gray;
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.scss:4]: Why is this needed? It looks like all the other icons use `bdl-gray-50` for their paths.

**Comment by jstoffan** [comment]
@MateuszMamczarz, it looks like the trash icons are different between the comment and annotation menus. Can we unify the icons and make them all the same color rather than as one-offs?

File: src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.scss (hunk 1 above)
Comment (MateuszMamczarz) [reply src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.scss:4]: All icons for comments are using `bdl-gray`, for annotations trash was different, but now it's the same as for comments. With this change this is how the whole menu looks like:
Screen Shot 2022-11-09 at 22 38 32

**Review by jstoffan — APPROVED** [review]
